src/burger/BurgerDetails.java

```diff
@@ -1,4 +1,55 @@
 package burger;
 
 public class BurgerDetails {
+
+    private String burgerName;
+    private String burgerTopping;
+    private double burgerPrice;
```

Comment:
variable burgerPrice should be priceOfTheBurger

Reply:
code is updated according to your direction